### PR TITLE
TEST: Update CI to compute and upload coverage for hi-ml-multimodal

### LINKED ISSUE #513: hi-ml-multimodal should be included in coverage reports
- [x] Create coverage reports from tests
- [x] Upload reports to codecov including the `flag` setting
- [x] Update `codecov.yml` to include a flag for `hi-ml-multimodal`

### PR DESCRIPTION
**Pull request by fepegar**
Resolves #513.

### CODE REVIEW

File: .github/workflows/multimodal-pr.yml [hunk 1]
```diff
@@ -105,9 +105,13 @@ jobs:
...
+          PIP_SOURCE="git+${REPO_URL}@${BRANCH_NAME}#${SUBDIR_STR}"
+          IN_NB_PATH="notebooks/phrase_grounding.ipynb"
+          OUT_NB_PATH="/tmp/phrase_grounding_output.ipynb"
           echo "The package will be installed from: $PIP_SOURCE"
           papermill \
             --parameters pip_source $PIP_SOURCE \
-            notebooks/phrase_grounding.ipynb \
-            /tmp/phrase_grounding_output.ipynb
+            $IN_NB_PATH \
+            $OUT_NB_PATH
```
Comment (fepegar) [inline .github/workflows/multimodal-pr.yml:117]: Unrelated changes for readability :)

File: hi-ml-multimodal/pytest.ini [hunk 2]
```diff
@@ -1,4 +1,4 @@
 [pytest]
 testpaths = test_multimodal
 log_cli = True
-log_cli_level = DEBUG
+log_cli_level = INFO
```
Comment (fepegar) [inline hi-ml-multimodal/pytest.ini:4]: To increase readability of logs.

**Review by mebristo — APPROVED** [review]

**Review by peterhessey — APPROVED** [review]
LGTM

**Review by ant0nsc — APPROVED** [review]
